Mathlib/Topology/Sheaves/SheafCondition/UniqueGluing.lean

```diff
@@ -84,23 +84,27 @@ We prove this to be equivalent to the usual one below in
 `TopCat.Presheaf.isSheaf_iff_isSheafUniqueGluing`
 -/
 def IsSheafUniqueGluing : Prop :=
-  ∀ ⦃ι : Type v⦄ (U : ι → Opens X) (sf : ∀ i : ι, F.obj (op (U i))),
-    IsCompatible F U sf → ∃! s : F.obj (op (iSup U)), IsGluing F U sf s
+  ∀ ⦃ι : Type w⦄ (U : ι → Opens X) (sf : ∀ i : ι, F.obj (op (U i))),
+    IsCompatible.{u, v, w} F U sf → ∃! s : F.obj (op (iSup U)), IsGluing.{u, v, w} F U sf s
```

Comment:
You didn't specify the `.{u, v, w}` in the mathlib3 PR.  I'll trust that you did it right.